src/inspector/processing/stats-processing.service.ts

```diff
@@ -29,19 +30,19 @@ export class StatsProcessingService implements OnModuleInit {
    * Calc stats by storage (validators info, att and prop duties) and push them to Prometheus
    */
   async calculateLidoStats(slot: bigint, possibleHighRewardValidators: string[]): Promise<void> {
-    const operators = await this.nodeOperatorsContract.getOperators();
+    const operators = await this.registryService.getOperators();
     this.prometheus.contractKeysTotal.set(
       { type: 'total' },
-      operators.reduce((sum, o) => sum + o.totalSigningKeys.toNumber(), 0),
+      operators.reduce((sum, o) => sum + o.totalSigningKeys, 0),
```

Comment:
`o.totalSigningKeys` is `number` now or just rely on type casting?

Reply:
It's `number` according to entity description